### PR TITLE
2.0.2 docs

### PR DESCRIPTION
**Pull request by brian-avery**
Created a release yaml for 2.0.2 and 2.0.1 since 2.0.1 was skipped. Also ran `make update-docs` to generate the new documentation.

### CODE REVIEW

File: data/releases/2.0.2.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2.0.1
```
Comment (brian-avery) [inline data/releases/2.0.2.yaml:1]: These values might be incorrect. The OSSM versions documentation was never updated for 2.0.2

**Comment by brian-avery** [comment]
This PR is blocked by: https://github.com/maistra/test-infra/pull/136

**Comment by brian-avery** [comment]
@rcernich @knrc can you please take a look?

**Review by rcernich — APPROVED** [review]

**Comment by brian-avery** [comment]
Thanks Rob

File: data/releases/2.0.2.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+maistraOperatorVersion: '2.0'
+maistraOperatorInstallableVersions: "v1.1, v2.0"
+maistraBranch: 'maistra-2.0'
+maistraURL: "https://maistra-2-0.maistra.io/"
+kialiVersion: 1.20.3
```
Comment (knrc) [inline data/releases/2.0.2.yaml]: 1.24.6

File: data/releases/2.0.1.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+version: 2.0.1
+maistraOperatorVersion: '2.0'
+maistraOperatorInstallableVersions: "v1.1, v2.0"
+maistraBranch: 'maistra-2.0'
+maistraURL: "https://maistra-2-0.maistra.io/"
+kialiVersion: 1.20.3
```
Comment (knrc) [inline data/releases/2.0.1.yaml]: 1.24.5

**Review by knrc — CHANGES_REQUESTED** [review]

File: data/releases/2.0.1.yaml (hunk 3 above)
Comment (brian-avery) [reply data/releases/2.0.1.yaml]: Thanks. Fixed.

**Review by knrc — APPROVED** [review]

**Pull request merged**